source/opt/instruction.h

```diff
@@ -228,6 +245,28 @@ inline void Instruction::ForEachInst(
   f(this);
 }
 
+inline void Instruction::ForEachInId(const std::function<void(uint32_t*)>& f) {
+  for (auto& opnd : operands_)
```

Comment:
Done.

Actually, there is now two versions of ForEachInId: a const version and one that allows for changing of the In Operands through the pointer. The const cannot be applied for the non-const version, but it can for the const version.